packages/module/patternfly-docs/content/extensions/chatbot/design-guidelines.md

```diff
@@ -57,6 +73,52 @@ A ChatBot can share a link to a [quick start](/extensions/quick-starts) that wil
 ![Bot message that links a quick start tile.](./img/chatbot-quickstarts-tile.svg)
 </div>
 
+### Message actions 
+
+To allow users to interact with bots messages, utilize message actions. Refer to [the message actions React example](/patternfly-ai/chatbot/messages#message-actions) for an interactive visual.
+
+The following actions can be used for some of the most common interactions:
+
+<div class="ws-docs-content-img">
+![.](./img/message-responses.svg)
+</div>
+
+1. **Feedback (good response):** Applies a positive rating to the message.
+1. **Feedback (bad response):** Applies a negative rating to the message.
+1. **Copy:** Copies the message content to the clipboard.
+1. **Share** 
+1. **Listen:** Reads the message content out loud.
+
+You can also use [custom message actions](/patternfly-ai/chatbot/messages#custom-message-actions) as needed for your particular use case. When using custom actions, be sure to add a tooltip that describes the effect that the action will have. For more information, view [our tooltips guidelines](/ux-writing/tooltips). 
+
+#### Message feedback
+
+A commonly used message response action is rating, which allows users to give feedback on the quality or helpfulness of a bot message. These actions include a thumbs-up icon, for a positive rating, and a thumbs-down icon, for a negative one:
+
+<div class="ws-docs-content-img">
+![.](./img/message-feedback.svg)
+</div>
+
+When users select either of these icons, you should present them with either:
+
+1. A thank-you card that confirms a user's response was received. Even if you don't use a feedback form, you should still display the thank-you card, so that users can be confident their rating went through.
+<br />
+<br />
+    <div class="ws-docs-content-img" style="width:75%">
+    ![.](./img/thank-you-card.svg)
+    </div>
+1. A feedback form that collects more details.
+<br />
+<br />
+    <div class="ws-docs-content-img" style="width:75%">
```

Comment:
Spacing is a little different around header area; see above. Not sure if we need to fix on dev side or design side.

Reply:
I believe this should match now -- Kayla pushed some Figma updates!